packages/app-desktop/splash.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JoplinSplashScreen</title>
+</head>
+<body>
+    <img src="./build/images/splash-image.svg" alt="joplin-icon">
```

Comment:
Will that scale depending on the container? Eg. if we make it 800x600 will it fill up the container?

Reply:
Yes it is scalable. I have checked and verified the same.